languages/elixir/exercises/concept/nil/docs/after.md

```diff
@@ -0,0 +1,110 @@
+[Nil][nil-dictionary] is an English word meaning "nothing" or "zero". In Elixir, `nil` is a special value that means an _absence_ of a value.
+
+```elixir
+# I do not have a favorite color
+favorite_color = nil
+```
+
+`nil` is an atom, but it is usually written as `nil`, not `:nil`.
+
+```elixir
+nil === :nil
+# => true
+```
+
+You can check if a variable's value is `nil` using [`==`][kernel-equal], with pattern matching, or using the [`is_nil` guard][kernel-is-nil].
+
+```elixir
+def call(phone_number) do
+  if phone_number == nil do
+    :error
+  end
+end
+```
+
+```elixir
+def call(phone_number) when is_nil(phone_number) do
+  :error
+end
+```
+
+```elixir
+def call(nil) do
+  :error
+end
+```
+
+## `if`
+
+Besides `cond`, Elixir also provides the macros [`if/2` and `unless/2`][getting-started-if-unless] which are useful when you need to check for only one condition.
+
+[`if/2`][kernel-if] accepts a condition and two options. It returns the first option if the condition is _truthy_, and the second option if the condition is _falsy_. [`unless/2`][kernel-unless] does the opposite.
+
+```elixir
+age = 15
+
+if age >= 16 do
+  "You are allowed to drink beer in Germany."
+else
+  "No beer for you!"
+end
+
+# => "No beer for you!"
+```
+
+If the second option is not given, `nil` will be returned.
+
+```elixir
+age = 15
+
+if age >= 16 do
+  "You are allowed to drink beer in Germany."
+end
+
+# => nil
+```
+
+It is also possible to write an `if` expression on a single line. Note the comma after the condition.
```

Comment:
Would it be useful to qualify the usage of a one-line `if` expression? E.g. "It is also possible to write an `if` expression on a single line _when_ used for short expressions."

I often find students on the v2 track try to use one-line syntax for everything and force multi-line functions into single line syntax.